src/sentry/static/sentry/less/base.less

```diff
@@ -1,3 +1,6 @@
+@font-family-base: 'Rubik', 'Avenir Next', 'Helvetica Neue', sans-serif;
+@font-family-code: 'IBM Plex', Consolas, 'Courier New', monospace;
+
```

Comment:
Consolidating the type variables into this file so that it’s obvious where all our base typography styles now live.